themes/vue/layout/partials/language_dropdown.ejs

```diff
@@ -7,6 +7,7 @@
     <li><a href="https://kr.vuejs.org/<%- page.path %>" class="nav-link" target="_blank" rel="noopener">한국어</a></li>
     <li><a href="https://br.vuejs.org/<%- page.path %>" class="nav-link" target="_blank" rel="noopener">Português</a></li>
     <li><a href="https://fr.vuejs.org/<%- page.path %>" class="nav-link" target="_blank" rel="noopener">Français</a></li>
+    <li><a href="https://es.vuejs.org/<%- page.path %>" class="nav-link" target="_blank" rel="noopener">Spanish</a></li>
```

Comment:
maybe the name should be in the native language instead of English like in the other options? (Español instead of Spanish)